velox/vector/DictionaryVector.h

```diff
@@ -214,6 +215,18 @@ class DictionaryVector : public SimpleVector<T> {
     setInternalState();
   }
 
+  void resize(vector_size_t size, bool setNotNull = true) override {
```

Comment:
Would you document this method to explain what values are used for the "added" elements?